R/plot.R

```diff
@@ -66,7 +68,7 @@ synthdid_plot = function(estimates, treated.name = 'treated', control.name = 'sy
   plot.descriptions = lapply(1:nrow(grid), function(row) {
     est = estimates[[grid$estimate[row]]]
     over = overlay[grid$overlay[row]]
-
+    se = sqrt(vcov(est, method=se.method))
```

Comment:
If you want to be super pedantic and cover even "Notes" from R cmd check:
```
synthdid_plot : <anonymous>: no visible global function definition for vcov
```
you can add @importFrom stats vcov like here https://github.com/grf-labs/grf/blob/master/r-package/grf/R/average_treatment_effect.R#L125

or just do: `stats::vcov`